src/proxy/show.go

```diff
@@ -39,6 +39,12 @@ func (spanner *Spanner) handleShowCreateDatabase(session *driver.Session, query
 	return spanner.ExecuteSingle(query)
 }
 
+// handleShowTablesWrapper
+func (spanner *Spanner) handleShowTablesWrapper(session *driver.Session, query string, node sqlparser.Statement) (*sqltypes.Result, error) {
```

Comment:
The handleShowTablesWrapper is for?  
Why we used handleShowTables instead ?

Reply:
Ok，I change the name.